raiden/tests/utils/mock_client.py

```diff
@@ -268,7 +272,7 @@ def manager_addresses(self):
             for manager in self.token_manager.values()
         ]
 
-    def tokenadded_filter(self):
+    def tokenadded_filter(self, from_block='', to_block=''):
```

Comment:
it looks odd to default `from_block` and `to_block` to a string, shouldn't it be `None` as default and `int` when setting?